libs/SalesforceReact/package.json

```diff
@@ -1,31 +1,38 @@
 {
-    "name": "SalesforceReact",
-    "version": "12.0.0",
-    "private": true,
-    "scripts": {
-        "start": "node node_modules/react-native/local-cli/cli.js start"
-    },
-    "dependencies": {
-        "create-react-class": "^15.7.0",
-        "react": "18.1.0",
-        "react-native": "0.70.14",
-        "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev"
-    },
-    "devDependencies": {
-        "@babel/core": "^7.12.9",
-        "@babel/runtime": "^7.12.5",
-        "@babel/preset-env": "^7.1.6",
-        "@react-native-community/eslint-config": "^2.0.0",
-        "@types/jest": "^26.0.24",
-        "@types/react": "^18.0.21",
-        "@types/react-native": "^0.70.14",
-        "@types/react-test-renderer": "^18.0.0",
-        "babel-jest": "^26.6.3",
-        "chai": "4.2.0",
-        "eslint": "^7.32.0",
-        "jest": "^26.6.3",
-        "metro-react-native-babel-preset": "^0.72.3",
-        "react-test-renderer": "18.1.0",
-        "typescript": "^4.8.3"
-    }
+  "name": "SalesforceReact",
+  "version": "12.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start"
+  },
+  "dependencies": {
```

Comment:
I used `ncu` to update the entire dependency list to current specs, which really seemed to help get a consistent configuration when compared to manually updating individual dependencies.